kolibri/core/tqdm_shim.py

```diff
@@ -0,0 +1,17 @@
+from __future__ import print_function
+try:
+    # import multiprocessing
+    import myhero
+except ImportError as e:
+    # import our shim
+    class tqdm(object):
+        def __init__(*args, **kwargs):
+            print("tqdm __init__ shim called!")
+
+        def update(*args, **kwargs):
+            print("tqdm update shim called!")
```

Comment:
Should change these to `logging.info` - see dev docs for how to import logging properly - maybe call with the value that was passed in, so that instead of a progress bar, we see the logged progress in the log?